types/types.go

```diff
@@ -378,6 +378,9 @@ type SystemContext struct {
 	// === dir.Transport overrides ===
 	// DirForceCompress compresses the image layers if set to true
 	DirForceCompress bool
+
+	// Optional cross-repository mount to use for Docker registry destionations.
+	DockerMount *reference.Named
```

Comment:
Nits:
- Please move this into the `=== docker.Transport overrides ===` section.
- `s/destionations/destinations/`